vendor/github.com/prometheus/alertmanager/cluster/cluster.go

```diff
@@ -584,8 +584,13 @@ func (p *Peer) Ready() bool {
 }
 
 // Wait until Settle() has finished.
```

Comment:
Shouldn’t `WaitReady` return `error`, so that client knows whether peer is now ready or not? WDYT?